### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Modules: Focus is not visible on all the controls.

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#177]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**WCAG ID Violated:** WCAG2.4.7

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #C+E-H; #.NETWebsites; #Chrome; #Edge; WCAG2.4.7

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Modules and press enter
6.	Validate whether focus is visible on all controls or not

**Actual Result:**
Focus is not visible on all the controls

**Expected Result:**
Focus should be visible on all the controls

**Note:**
This issue is observed in all the pages
1.Webserver->Response Compression
2.Webserver->Request Filtering
3.Webserver->Response Headers
4.Webserver->Request Tracing
5.Webserver->Static Content
6.Webserver->Url Rewrite

**User Impact:**
Keyboard only users find it difficult in navigating through the application if focus is not visible

**Attaching screenshots for your reference:**
![bug 177_focus visible_url rewrite](https://user-images.githubusercontent.com/42638744/44835734-b6dc8080-ac52-11e8-9917-51ff542f1e43.png)
![bug 177_focus visible_static content](https://user-images.githubusercontent.com/42638744/44835747-bfcd5200-ac52-11e8-9434-1795e328b8b3.png)
![bug 177_focus visible_response headers](https://user-images.githubusercontent.com/42638744/44835761-c8be2380-ac52-11e8-81c0-b2bd8fba4a82.png)
![bug 177_request tracing_focus visible](https://user-images.githubusercontent.com/42638744/44835809-e55a5b80-ac52-11e8-9f9b-9c148d5ad40a.png)
![bug 177_focus visible_response compression](https://user-images.githubusercontent.com/42638744/44835831-f014f080-ac52-11e8-9e88-a614d8a85bad.png)
![bug 177_focus visible_request filtering](https://user-images.githubusercontent.com/42638744/44835848-f905c200-ac52-11e8-8702-4aac0cf7e8fa.png)
![bug 177_focus visible](https://user-images.githubusercontent.com/42638744/44835858-ff943980-ac52-11e8-88e5-20b8251cec4d.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
